modules/custom/entity_access_by_field/entity_access_by_field.install

```diff
@@ -100,3 +100,10 @@ function _entity_access_by_field_get_permissions($role) {
   }
   return [];
 }
+
+/**
+ * Rebuild node access.
+ */
+function entity_access_by_field_update_8004() {
```

Comment:
Please move it to post_update_hook_name.

Reply:
I have moved it.